### ISSUE TITLE
Create a Smoke test

### ISSUE THREAD

**Issue by lakruzz** [lakruzz/SuperheltV5#3]
Add a test that can be used after deployment to verify that both database and app is running and is accessible

**Comment by lakruzz** [comment]
I added a script that can take a MySql config file `my.cnf` as argument and test if app and database is running - I've also added two config files - all is stoed in `src/mysql` directory

Closed with b5e6e8e0f04ae767538263b7a8607b4c838f6d08

**Comment by lakruzz** [comment]
```
Usage: smoketest.sh <defaults-file>
```

To test dev:

```shell
src/test/smoketest/smoketest.sh src/mysql/my.dev.cnf
```

To test prod:
```shell
src/test/smoketest/smoketest.sh src/mysql/my.prod.cnf
```

You can run the script without the required argument to learn more:

```shell
src/test/smoketest/smoketest.sh
```
